src/KW_radio.h

```diff
@@ -31,19 +31,53 @@
 extern "C" {
 #endif
 
+/*
```

Comment:
If this is intended to be a doxygen documentation, you should add double-asterisks to the first line or else doxygen will not see it